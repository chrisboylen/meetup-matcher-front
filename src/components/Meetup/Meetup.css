.meetup {
  background-color: #f7eeeeba;
  background-size: cover;
  border-radius: 10px;
  box-shadow: 0px 0px 22px 3px rgba(43, 38, 38, 0.91);
  color: #280303;
  margin: 0 0 1rem 0;
  padding: 0 0 .2rem 0; }
  .meetup h4, .meetup a {
    margin: 1.5rem; }
  .meetup h2 {
    background-color: #fff5f5c7;
    border-radius: 12px 12px 0 0;
    margin: 0;
    padding: .5rem;
    text-align: center; }
  .meetup span {
    color: #494545;
    margin: 0 0 0 1rem; }
  .meetup a {
    color: #ff0000;
    display: block; }
