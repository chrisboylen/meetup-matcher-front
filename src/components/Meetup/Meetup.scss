$mainWhite: #fef5f5;
$mainDark: #280303;

.meetup {
  background-color: #f7eeeed1;
  border-radius: 10px;
  color: $mainDark;
  margin: 0 0 1rem 0;
  padding: 1rem;
    h2 {
      margin: 0;
      text-align: center;
    }
    a {
      display: block;
    }
}