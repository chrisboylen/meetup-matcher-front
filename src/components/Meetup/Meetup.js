import React from 'react';
import PropTypes from 'prop-types';
import './Meetup.css';

export const Meetup = ({...meetup}) => {
  return (
    <article className="meetup">
      <h2>{meetup.name.name}</h2>
      {meetup.description}
      <a href={meetup.website}>website</a>
    </article>
  );
};

Meetup.propTypes = {
  meetup: PropTypes.object
};