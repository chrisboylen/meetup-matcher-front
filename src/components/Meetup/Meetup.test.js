import React from 'react';
import { shallow } from 'enzyme';
import { Meetup } from './Meetup';

describe('Meetup', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Meetup name='bingo'/>);

    expect(wrapper).toMatchSnapshot();
  });
});