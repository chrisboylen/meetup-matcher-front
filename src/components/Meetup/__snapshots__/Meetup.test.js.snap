// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meetup should match snapshot 1`] = `
<article
  className="meetup"
>
  <h2 />
  <a>
    website
  </a>
</article>
`;
